middlewares/hacker.middleware.js

```diff
@@ -562,6 +547,8 @@ async function getStats(req, res, next) {
 module.exports = {
     parsePatch: parsePatch,
```

Comment:
Do we still need a parsePatch within hacker.middleware?

Reply:
It was already there but it seems like the only goal is to delete an Id, I can put this in parsePatch